File: src/main/java/com/undsf/mcmodmgr/McModManagerApplication.java
package com.undsf.mcmodmgr;

import org.mybatis.spring.annotation.MapperScan;
import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;

@SpringBootApplication
@MapperScan("com.undsf.mcmodmgr.mappers")
public class McModManagerApplication {
    public static void main(String[] args) {
        SpringApplication.run(McModManagerApplication.class, args);
    }
}

File: src/main/kotlin/com/undsf/mcmodmgr/McModManagerApplication.kt
package com.undsf.mcmodmgr

import org.mybatis.spring.annotation.MapperScan
import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication

@SpringBootApplication
@MapperScan("com.undsf.mcmodmgr.mappers")
class McModManagerApplication

fun main(args: Array<String>) {
    runApplication<McModManagerApplication>(*args)
}
